src/test/java/com/assignmentTest/Assignment9Test.java

```diff
@@ -0,0 +1,37 @@
+package com.assignmentTest;
+
+import com.assignment.Assignment9;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Assignment9Test {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void testEvenNumbers() {
+        int min = 1, max = 500;
+        ArrayList<Integer> evenNumbersVerification = new ArrayList<>();
+        Assignment9 assignment9 = new Assignment9();
+
+        ArrayList evenNumbers = assignment9.getEvenNumber(min, max);
```

Comment:
As you are using reference one time for class objectsss Assignment9 
     ArrayList evenNumbers = new Assignment9().getEvenNumber(min, max);